main/streams/xp_socket.c

```diff
@@ -869,7 +868,7 @@ static inline int php_tcp_sockop_accept(php_stream *stream, php_netstream_data_t
 		nodelay = 1;
 	}
 
-	clisock = php_network_accept_incoming(sock->socket,
+	php_socket_t clisock = php_network_accept_incoming(sock->socket,
```

Comment:
@Girgias This is a regression on Windows where error check no longer works. Please check carefully things and the whole context before changing the types. Also it would be good not to mix so many things together as it's pretty much impossible to catch this during the review. See GH-19798 and the fix in GH-19881 for more details.